### ISSUE TITLE
Publish updates to npm

### ISSUE THREAD

**Issue by wileyj** [stacks-network/stacks-test-tools#6]
Ok indeed it looks like neither of us can publish. Could be nice to have it in the CI

_Originally posted by @hugocaillard in https://github.com/stacks-network/stacks-test-tools/issues/4#issuecomment-2033942980_

**Comment by friedger** [comment]
Also see #15

**Comment by wileyj** [comment]
> Also see [#15](https://github.com/stacks-network/clarunit/pull/15)

Agreed. spoke about this with @BowTiedDevOps and @moodmosaic yesterday - we think the process used in rendezvous would also work here, but for now this will be a manual push. i'll get it done

**Comment by wileyj** [comment]
https://github.com/stacks-network/clarunit/pull/17
